mappings/mappings.php

```diff
@@ -8,741 +8,1196 @@
             '008'                                 => '80legscrawler',
             '2345chrome'                          => '2345browser',
             '2345explorer'                        => '2345browser',
-            '80legs'                              => '80legscrawler',
+            '360browser'                          => '360phonebrowser',
+            '7720wabc'                            => '77wabc',
             '80bot'                               => '80legscrawler',
+            '80legs'                              => '80legscrawler',
             'aboundex'                            => 'aboundexbot',
             'accoonabot'                          => 'accoonabizagent',
+            'addthiscom'                          => 'addthiscomrobot',
+            'admantx'                             => 'admantxplatformsemanticanalyzer',
+            'adsbotbot'                           => 'adsbotgoogle',
             'ahrefahrefsbot'                      => 'ahrefsbot',
+            'amigaaweb'                           => 'amiga',
             'android'                             => 'androidbrowser',
+            'androidchrome'                       => 'chrome',
             'androidwebkit'                       => 'androidbrowser',
             'antiplagspider'                      => 'dawinciantiplagspider',
             'antnutch'                            => 'nutch',
             'aol'                                 => 'aoldesktop',
+            'apachehttpd'                         => 'apachehttpclient',
+            'apachejakarta'                       => 'jakartacommonshttpclient',
+            'apachenutch'                         => 'nutch',
             'appenginegoogle'                     => 'googleappengine',
+            'applewebkit'                         => null,
```

Comment:
Shouldn't `null` be reserved for values that represent the "unknown" detection (like `Default Browser` in browscap)?

Reply:
`applewebkit` is not a browser. I mapped everything which is not a browser to null (something like `ios` or `false`).

I use this project to see, browscap works fine or not. I dont think ist helpful to flood the result with errors because another parser is mixing engines into browser or devices into platforms.

Reply:
Shall I add comments which parser gives this result?

Reply:
That makes sense, I'm not sure if we need to know which parser has that result, but perhaps it would be nice to know why it's being marked as `null` (if not obvious).  I've added inline comments to a couple that I think probably shouldn't be `null` but feel free to chime in why they should be. :)

Reply:
removed this mapping